docker_images/diffusers/tests/test_api_text_to_image.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
Please add a test template to common as well

Reply:
It's already there...